server/api-manager.js

```diff
@@ -15,12 +15,13 @@ const URL_SALE = BASE + "sale";
 const URL_STAFF = BASE + "staff";
 const URL_ENROLLMENT = BASE + "enrollment";
 const URL_APPOINTMENT = BASE + "appointment";
-const APIKEY = "76af57a017f64fcd9fc16cc5032404a0";
+//const APIKEY = "76af57a017f64fcd9fc16cc5032404a0";
+const APIKEY = "7db287c206374b2f911ddc918879983d";
 const SITEID = "-99";
 
 import MindbodyRequest from "./requests.js";
 import QueryString from "query-string";
-export default class MindbodyQueries {
+export class MindbodyQueries {
```

Comment:
There were two default exports in this file, and that made node reject the file. I think we intend only for the one-and-only MindbodyAccess object to be exported.